Lectures/Lecture3_BigCollaborations.md

```diff
@@ -1,41 +1,62 @@
 
 ## Vision of outcome
-__What skills and capabilities will the course participants obtain after finishing this lecture and the corresponding assignments?__
+- Course participants appreciate the need for and the benefits of large collaborations for many driving questions in modern science. 
+- Course participants have critically reflected on challenges faced by large collaborations and choices to be made in their structure, organization, and membership.
+- Course participants have investigated examples of working within large collaborations, in particular the credit structures and  ways to promote junior researchers. 
 
 ## Embedding/community
-__please mention the main community that should be considered when studying the topics of this lecture, e.g. individual researchers, research community, employers, general public, etc. __
+This lecture focuses on large national as well as international collaborations. Many of the large collaborations involve interdisciplinary components, both within physics and with engineering and industry. See https://bigscience.nl/about for a concerted effort in NL to foster synergies between physics and technology in large collaborations. 
 
 ## Methods/exercises
-**Topics covered in lecture**
-+ topic 1
-+ topic 2
++ In preparation for this lecture, course participants are asked to watch the two-part video lecture by Dr Niels Martens on the epistemology of scientific collaborations. Lecture 1 of this set (https://www.youtube.com/watch?v=M1giIxeLuIg) discusses examples of 'big science' projects such as CERN and the Next Generation Event Horizon Telescope. Lecture 2 of this set (https://www.youtube.com/watch?v=DRt1uTytWf8) focuses on ways to manage the large number of co-authors that are involved in these projects and policies for giving appropriate credit. 
++ Course participants will work in small groups to investigate the credit structure of one collaboration they were assigned from the list below. They prepare a short presentation to their peers. 
++ The groups will give their presentations to the class and have a final critical discussion on the topic of large collaborations, for which Dr. Niels Martens will also join. 
 
 ## Evaluation
-__assignments or evaluation of the desired learning outcome__
++ Students answer the quiz about the video lectures by Dr. Niels Martens. 
```

Comment:
the quiz questions are about the podcast, not the videos.

Reply:
corrected this in the updated version